### DISCUSSION TITLE
构建 Docker 镜像让使用者有统一的编译环境

### DISCUSSION THREAD

**Discussion by imfing** [whutug/whu-thesis#43] [Ideas]
刚开始做这个项目的时候就有考虑过构建一个 Docker 镜像来统一编译环境，让使用者以及 GitHub Action 编译结果（特别是字体）有着一致的显示效果。
Workflow 中使用的 `xu-cheng/latex-action@v2` 编译后的文件字体还是 Ubuntu 系默认字体。
如果可以的话，新开一个 Repo，将 Windows 系字体传上去，在 Docker 镜像中安装这些字体。最后默认使用这些字体编译，应该能达到和 Windows 一致的编译效果。

参考：
* https://github.com/xu-cheng/latex-docker

**Comment by tanukihee** [comment]
中易字体有版权的，也许可以试试方正？
